pydkron/client.py

```diff
@@ -128,7 +128,25 @@ def get_executions(self, name):
         """
         Get the job executions for a named job
         """
-        resp = self._call(_GET, "/executions/%s" % name)
```

Comment:
The url I found to get executions of job is /v1/jobs/<job-name>/executions/.

So I change both code and tests.

FYI.